o2a/mappers/action_mapper.py

```diff
@@ -39,20 +44,21 @@ def __init__(self, oozie_node: Element, name: str, dag_name: str, props: Propert
 
     def on_parse_node(self, workflow: Workflow):
         super().on_parse_node(workflow)
-        self._parse_config()
+        self._parse_config(workflow)
+
+    def _parse_config(self, workflow: Workflow):
+        job_xml_nodes = find_nodes_by_tag(self.oozie_node, TAG_JOB_XML)
+        new_properties = extract_properties_from_job_xml_nodes(
+            job_xml_nodes=job_xml_nodes, input_directory_path=workflow.input_directory_path, props=self.props
+        )
+        self.props.action_node_properties.update(new_properties)
 
-    def _parse_config(self):
-        action_node_properties: Dict[str, str] = {}
-        config = self.oozie_node.find("configuration")
-        if config:
-            props = self.props
-            property_nodes = xml_utils.find_nodes_by_tag(config, "property")
-            if property_nodes:
-                for node in property_nodes:
-                    name = node.find("name").text
-                    value = el_utils.replace_el_with_var(node.find("value").text, props=props, quote=False)
-                    action_node_properties[name] = value
-        self.props.action_node_properties = action_node_properties
+        configuration_node = find_node_by_tag(self.oozie_node, TAG_CONFIGURATION)
+        if configuration_node is not None:
+            new_properties = extract_properties_from_configuration_node(
```

Comment:
`new_properties` -> `conf_node_properties` ?

Reply:
I updated the PR.